index.js

```diff
@@ -43,7 +43,9 @@ module.exports = {
     "function-comma-space-after": "always-single-line",
     "function-comma-space-before": "never",
     "function-max-empty-lines": 0,
-    "function-name-case": "lower",
+    "function-name-case": ["lower", {
+        "ignoreFunctions": ["${unsafeCSS"]
```

Comment:
What's the use case for using `unsafeCSS` in Rubrics?  Maybe there is another way to accomplish what you need without using it?  I'm not against amending this rule - just that unsafeCSS is a potential security risk and ideally we avoid using it.

Reply:
We are using it to inline an SVG as a background image instead of having to deploy it and use a URL. The SVG code is just a base64 data uri generated by us.

Reply:
We may be able to get rid of it after removing the dnd sortable stuff in favour of the new list component but that isn't going to happen now. For now I've just added this ru8le to that repo if we'd rather just proceed with that in the hopes it removed we can ignore this PR.

Reply:
I think it's a valid directive to support so I don't think we necessarily want the linting to prevent someone from using it.

FWIW, in core we just bake it in: https://github.com/BrightspaceUI/core/blob/main/components/skeleton/skeleton-mixin.js#L42

Reply:
Fair, we could bake it in like that. We had it separate so we could regenerate it if it ever changes (bit more data driven) but this resource is unlikely to change.